media/css/firefox/nothing-personal/_things.scss

```diff
@@ -0,0 +1,338 @@
+    }
+}
+
+// hide details if JS available
+.c-things-list-container {
+    display: none;
+}
+
+.c-things-list {
+    background-color: var(--background-color);
+    border: $border-black;
+    padding: $spacing-md;
+
+    &.mzp-u-list-styled {
+        margin-left: 0;
+        padding-left: $spacing-xl;
+    }
+}
+
+.c-things-browser-fox-bush-container {
+    position: relative;
+}
+
+$fox-bush-width: 136px;
+$fox-bush-height: 87px;
+
+.c-things-fox-bush {
+    display: none;
+
+    @media #{$mq-lg} {
+        display: block;
+        position: absolute;
+        left: calc(#{$layout-lg} * -2);
+        top: $layout-lg;
+
+        &::after {
+            content: '';
+            position: absolute;
+            background-color: #0EBF45;
+            height: $fox-bush-height;
+            width: $fox-bush-width;
+            left: 10px;
+            top: 10px;
+            border: $border-black;
+        }
+    }
+}
+
+.c-things-fox-bush-image {
+    height: $fox-bush-height;
+    width: $fox-bush-width;
+    position: relative;
+    z-index: 10;
+
+    // static image has baked-in border, but doesn't need any cropping
+    @media (prefers-reduced-motion: no-preference) {
+        border: 1px solid $page-background;
+        object-fit: cover;
+        object-position: bottom; // crop out browser bar at top
+    }
+}
+
+.c-things-browser {
+    position: relative;
+}
+
+.c-things-foxy-sticker {
+    position: absolute;
+    right: -10px;
+    top: -50px;
+    width: 127px;
+    height: 119px;
+
+    @media #{$mq-md} {
+        right: -70px;
+        top: 30px;
+        height: 145px;
+        width: 155px;
+    }
+}
+
+.c-things-smiley-sticker {
+    position: absolute;
+    left: 50px;
+    bottom: -70px;
+    width: 91px;
+    height: 89px;
+
+    @media #{$mq-md} {
+        left: 75px;
+        width: 104px;
+        height: 102px;
+    }
+}
+
+.c-things-heart-sticker {
+    position: absolute;
+    left: 10px;
+    bottom: -70px;
+    width: 62px;
+    height: 57px;
+
+    @media #{$mq-md} {
+        left: 30px;
+        width: 72px;
+        height: 66px;
+    }
+}
+
+.c-things-output-container {
+    background-color: var(--background-color);
+    border: $border-black;
+    margin-bottom: $layout-sm; // match title margin
+    padding: $spacing-md;
+    min-height: 3lh; // preserve space for longest line length responses
+
+    @media #{$mq-sm} {
+        min-height: 2lh;
+    }
+}
+
+.c-things-output {
+    margin-bottom: 0;
+    text-wrap: pretty;
+}
+
+.c-things-button-container {
+    text-align: center;
+}
+
+@keyframes shine {
+    0%{
+        background-position: 0 0;
+    }
+
+    100% {
+        background-position: 100% 0;
+    }
+}
+
+@keyframes pulse {
+    0% {
+        scale: 1;
+    }
+
+    5% {
+        scale: 1.05;
+    }
+
+    10% {
+        scale: 1;
+    }
+
+    15% {
+        scale: 1.025;
+    }
+
+    20% {
+        scale: 1;
+    }
+
+    100% {
+        scale: 1;
+    }
+}
+
+.c-things-button {
```

Comment:
Should this button also get `cursor: pointer;` to help hint at clickability? (Arguably the finger should only be for links but we do apply it on other non-link interactive buttons like form submits, so maybe it's universally understood as "I can click this"?)

Reply:
I like the pointer, does seem to help indicate clickability